ctapipe/core/tests/test_traits.py

```diff
@@ -433,6 +433,33 @@ class SomeComponent(TelescopeComponent):
     assert config["SomeComponent"]["tel_param1"] == [("type", "*", 6.0)]
 
 
+def test_telescope_parameter_from_cli(mock_subarray):
+    """
+    Test we can pass single default for telescope components via cli
+    see #1559
+    """
+
+    from ctapipe.core import Tool, run_tool
+
+    class SomeComponent(TelescopeComponent):
+        path = TelescopeParameter(Path(), default_value=None).tag(config=True)
+        val = TelescopeParameter(Float(), default_value=1.0).tag(config=True)
+
+    class TelescopeTool(Tool):
+        def setup(self):
+            self.comp = SomeComponent(subarray=mock_subarray, parent=self)
+
+    tool = TelescopeTool()
+    run_tool(tool)
+    assert tool.comp.path == [("type", "*", None)]
+    assert tool.comp.val == [("type", "*", 1.0)]
+
+    tool = TelescopeTool()
```

Comment:
why the re-instantiation of the tool?
wouldn't the new run overwrite previous traits?

Reply:
I want to be sure that I have a fresh instance and the run before does not interfere. This is about testing the cli and when using Tools, you get a fresh one with each run.